contrib/containers/guix/scripts/guix-start

```diff
@@ -17,15 +13,27 @@ XCODE_VERSION="12.2"
 XCODE_RELEASE="12B45b"
 XCODE_ARCHIVE="Xcode-${XCODE_VERSION}-${XCODE_RELEASE}-extracted-SDK-with-libcxx-headers"
 
+export SDK_PATH="${SDK_PATH:-${WORKSPACE_PATH}/depends/SDKs}"
+
 # Check if macOS SDK is present, if not, download it
-if [ ! -d "${WORKSPACE_PATH}/depends/SDKs/${XCODE_ARCHIVE}" ]
-then
-    mkdir -p "${WORKSPACE_PATH}/depends/SDKs"
-    curl -L https://bitcoincore.org/depends-sources/sdks/${XCODE_ARCHIVE}.tar.gz | tar -xz -C "${WORKSPACE_PATH}/depends/SDKs"
+if [[ ! -d "${SDK_PATH}/${XCODE_ARCHIVE}" ]]; then
+    echo "Preparing macOS SDK..."
+    mkdir -p "${SDK_PATH}"
+    curl -L https://bitcoincore.org/depends-sources/sdks/${XCODE_ARCHIVE}.tar.gz | tar -xz -C "${SDK_PATH}"
```

Comment:
Consider

```bash
XCODE_ARCHIVE="Xcode-${XCODE_VERSION}-${XCODE_RELEASE}-extracted-SDK-with-libcxx-headers.tar.gz"
XCODE_SOURCE="${XCODE_SOURCE:-https://bitcoincore.org/depends-sources/sdks}"

if [[ ! -d "${SDK_PATH}/${XCODE_ARCHIVE}" ]]; then
    echo "Preparing macOS SDK..."
    mkdir -p "${SDK_PATH}"
    curl -L "${XCODE_SOURCE}/${XCODE_ARCHIVE}" | tar -xz -C "${SDK_PATH}"
fi
```

Should let us point to Dash's servers in the future (using Bitcoin's servers now because our servers don't have Xcode 15, needed by an upcoming backport)

Reply:
Resolved by 07f056a